site/docs/components/slider.mdx

```diff
@@ -28,14 +28,21 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
 
 <iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FeZKEE5kXbEMY3lx84oz8iN%2F%25E2%259D%25A4%25EF%25B8%258F-UI-Kit-Heart%3Fnode-id%3D16080%253A4049" allowfullscreen></iframe>
 
+## Options
+
+- **Default:**
+    - In a default slider, the user can make an imprecise selection from a range of values, where there is no implication of progression from one side to the other.
+- **Progress:**
+    - In a progress slider, the user can make an precise selection from a range of values, aided by an exact indication of value. A sense of progress is implied, from one side to the other.
```

Comment:
it's not super clear to me why in one of these cases the selection is precise and in the other imprecise? Is this because the handle is imprecise, but the current value field enables the exact current value to be set?

Reply:
Yeah, that was the intention, but I definitely see that this could be confusing. Maybe I'm just making this more difficult than it needs to be; I can remove the words precise/imprecise and that could be better.

Reply:
Have removed the precise/imprecise thing all together - @dlimiter and/or @katsambarcus, is that better?